packages/storybook-ui/src/modules/ui/components/assets/svg_package.js

```diff
@@ -0,0 +1,175 @@
+/* eslint-disable */
+(function webpackUniversalModuleDefinition(root, factory) {
```

Comment:
I'm not following why we're committing a webpack generated module into the repo with svgs in base64 string format.

I think for maintainability sake, we should include the original svgs in the repo and generate them during runtime.